man/import.q.sorts.Rd

```diff
@@ -17,69 +18,75 @@
 	}
 	\item{conditions}{
 		A character vector of study conditions, such as \code{c("before","after")}, same as folders under \code{q.sorts.dir}.
-		Defaults to NULL in which case there are is only condition, and \code{*.csv}s are directly under \code{q.sorts.dir}.
+		Defaults to \code{NULL} in which case there is only condition, and CSVs are expected directly under \code{q.sorts.dir}.
 	}
 	\item{manual.lookup}{
-		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
-		Defaults to NULL in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
+		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings (ID) in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
+		Defaults to \code{NULL} in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
 	}
 }
 
 \details{
-Q-sorts in their raw form, as created by participants, have columns as ranks (from, say, -6 to +6) with cards (items) sorted in rows.
-The vertical dimension of q-sorts is \emph{meaningless}.
-For reproduceable research, q-sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
+	Q sorts in their raw form, as created by participants, have columns as ranks (from, say, \code{-6} to \code{+6}) with cards (items) sorted in rows.
+	The vertical dimension of Q sorts is \emph{meaningless}.
+	For reproduceable research, Q sorts, as all data, should be maintained in their rawest form and any processing should be well-documented.
 
-That is what this function offers, by reading in q-sorts from the form in which they were entered by participants.
+	That is what this function offers, by reading in Q sorts from the form in which they were entered by participants.
 
-Q-sorts are best entered not by typing up the full form of an item, but some unique string printed on the card.
-So as \emph{not} to influence participants, these strings should be meaningless to humans.
-This function offers either a set of arbitrary strings hard entered by a researcher for identification in \code{manual.lookup}, or expects a cryptographic hash of the complete item wording, as conveniently also created by \code{\link{make.cards}}.
-These hash values change whenever \emph{anything} in the full item wordings is changed, and allow a precise idenfication of different versions of an item.
+	Q sorts are best entered not by typing up the full form of an item, but some unique string (ID) printed on the card.
+	So as \emph{not} to influence participants, these strings should be meaningless to humans.
```

Comment:
Suggestion for clarity:

This function imports Q sorts from their raw format stored in *.CSV format, in the form in which they were sorted by participants.

Q sorts in their raw form have columns as ranks (from, say, -6 to +6) with cards (items) sorted in rows. The vertical dimension of Q sorts is meaningless. It is recommended that Q sort data are kept in their rawest form (with clear documentation of any processing applied on this data). This is good practice for reproducible research.

Q sorts are best entered using a unique string (ID) printed on the card, rather than by typing up the full form of an item. In order to avoid influencing participants, these strings should be meaningless to humans.